chromadb/telemetry/opentelemetry/__init__.py

```diff
@@ -120,17 +121,32 @@ def trace_method(
     """A decorator that traces a method."""
 
     def decorator(f: Callable[..., Any]) -> Callable[..., Any]:
-        @wraps(f)
-        def wrapper(*args: Any, **kwargs: Dict[Any, Any]) -> Any:
-            global tracer, granularity
-            if trace_granularity < granularity:
-                return f(*args, **kwargs)
-            if not tracer:
-                return f(*args, **kwargs)
-            with tracer.start_as_current_span(trace_name, attributes=attributes):
-                return f(*args, **kwargs)
-
-        return wrapper
+        if asyncio.iscoroutinefunction(f):
+
```

Comment:
Can we fix the spacing here? It's funky to have an empty line after an if-statement

Reply:
Interesting, I am sure I have fixed this and ran `pre-commit run`  which must have resulted in this formatting again.